tox.ini

```diff
@@ -48,7 +48,19 @@ deps =
     .[base,dev]
 commands =
     ; Run Unit-Test with coverage report.
-    pytest tests/unit \
+    pytest -m "not heavy" tests/unit \
+        --cov=otx \
+        --cov-report=xml:{toxworkdir}/coverage_{envname}.xml \
+        --cov-report=term-missing \
+        --cov-fail-under=0 \
+        {posargs}
+
+[testenv:heavy-unit-test-{py310, py311}]
+deps =
+    .[base,dev]
+commands =
+    ; Run Unit-Test with coverage report.
+    pytest -m "heavy" tests/unit \
```

Comment:
Does heavy mean resource intensive? Is `heavy` common term for such tests? I feel `resource-intensive` or something similar would be more descriptive and common?

Reply:
Also, this brings another question: why these tests are resource-intensive? Like the nature of the architecture or something else? Can it be simplified or converted into an integration test?

Reply:
I think this is a project wise issue from pytest and not an issue particularly from this PR. Evgeny will be looking this issue further.